src/Api/EidEasyApi.php

```diff
@@ -178,6 +178,43 @@ public function prepareFiles(array $files, array $parameters = []): array
         return $this->sendRequest('/api/signatures/prepare-files-for-signing', $data);
     }
 
+    /**
+     * @param string $file
+     * @param array $parameters
+     * @return string[]
+     */
+    public function prepareSignedFile(string $file, array $parameters = []): array
```

Comment:
Function name is not distinguishable from the "prepareFiles". Name it to prepareAddSignatureAsice
Avoid code duplication. Especially here it is likely that some parameters will be added.

Reply:
fixed